automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -54,6 +55,7 @@ object Leonardo extends WorkbenchClient with LazyLogging {
         operationName,
         ClusterStatus.withName(status),
         hostIp,
+        creator,
```

Comment:
Think you need to update https://github.com/broadinstitute/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala#L51 as well.

(All this model duplication makes me 😭 )

Reply:
oh, that's where that is. i ctrl+clicked to find it from ClusterKluge and it didn't work because i didn't have the automation project open separately. bah.